en/docs/design-the-project/design-using-the-architecture-view.md

```diff
@@ -1,10 +1,10 @@
-# Design using the Architecture Diagram
+# Design using the architecture view
```

Comment:
If `Architecture View` is the UI component why using lower case letters? In the following paragraph it has upper case letters which his inconsistent.

Reply:
Fixed.